src/tree/index.js

```diff
@@ -128,12 +128,18 @@ class Tree extends Component {
   }
 
   getAriaAttributes = () => {
-    const { readOnly, mode } = this.props
-    const attributes = {}
+    const { mode } = this.props
+
+    const attributes = {
+      /* https://www.w3.org/TR/wai-aria-1.1/#select
+       * https://www.w3.org/TR/wai-aria-1.1/#tree */
+      role: mode === 'simpleSelect' ? 'listbox' : 'tree',
+    }
+
+    if (/multiSelect|hierarchical/.test(mode)) {
+      attributes['aria-multiselectable'] = true
```

Comment:
The reason for 'true' : 'false' before is because that forces the values to be outputted, not only on true (boolean true).

Not sure if we should keep that (does not seem to cause any a11y-errors when running tests)
```
    return {
      /* https://www.w3.org/TR/wai-aria-1.1/#select
       * https://www.w3.org/TR/wai-aria-1.1/#tree */
      role: mode === 'simpleSelect' ? 'listbox' : 'tree',
      'aria-multiselectable': /multiSelect|hierarchical/.test(mode) ? 'true' : 'false',
    }
```

Reply:
I guess this needs revisiting. It was complaining about them earlier but that could have been due to something else.

Reply:
We need them to be strings for them to be emitted on false also.

Reply:
I think the reason a11y tests pass is because of this:

from https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

> **aria-multiselectable**
Include and set to true if the user can select more than one option. If set to true, every option should have a the aria-selected attribute included. If false or omitted, only the currently selected option, if any option is selected, needs the aria-selected attribute, and it must be set to true.

So going by this, it's ok to omit for single select cases.